deployments/s3/variables.tf

```diff
@@ -47,6 +47,12 @@ variable "lambda_supplies_key" {
   default = "supplies.zip"
 }
 
+variable "lambda_send_sms_key" {
+  description = "S3 Object key for SendSms lambda"
+  type = string
+  default = "send-sms.zip"
+}
```

Comment:
That var needs to be passed to the module through the `main.tf` otherwise it's useless to declare it here